launch/launch/actions/execute_process.py

```diff
@@ -71,15 +71,15 @@ class ExecuteProcess(Action):
 
     def __init__(
         self, *,
-        cmd: Iterable[SomeSubstitutionsType],
-        cwd: Optional[SomeSubstitutionsType] = None,
-        env: Optional[Dict[SomeSubstitutionsType, SomeSubstitutionsType]] = None,
-        shell: bool = False,
-        sigterm_timeout: SomeSubstitutionsType = LaunchConfiguration('sigterm_timeout', default=5),
-        sigkill_timeout: SomeSubstitutionsType = LaunchConfiguration('sigkill_timeout', default=5),
-        prefix: Optional[SomeSubstitutionsType] = None,
-        output: Optional[Text] = None,
-        log_cmd: bool = False,
```

Comment:
I thought these kinds worked in Python3.5?

Reply:
oh, maybe I went overkill. I'll check this afternoon.